app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -273,6 +274,7 @@ private void initializeDatabase() throws IOException {
     private void initializeContacts() throws IOException {
         final File file = new File(this.contactsPath);
         if (!file.exists()) {
+            LOGGER.warn("Database didn't exist -> created a new one");
```

Comment:
Either move this log to the end of the if-statement or change `created a new one` to `creating a new one`. Also change `Database didn't exist` to `Contacts part of the database didn't exist` or move the log to the `initializeDatabase` method.